contracts/deploy_tools/usage.md

```diff
@@ -15,20 +10,20 @@ Usage: tl-deploy [OPTIONS] COMMAND [ARGS]...
   Commandline tool to deploy the Trustlines contracts
 
 Options:
-  --help  Show this message and exit.
+  --version  Prints the version of the software
+  --help     Show this message and exit.
 
 Commands:
-  currencynetwork           Deploy a currency network contract.
-  exchange                  Deploy an exchange contract.
-  identity-implementation   Deploy an identity implementation contract.
-  identity-proxy-factory    Deploy an identity proxy factory.
-  test                      Deploy contracts for testing.
+  currencynetwork          Deploy a currency network contract.
+  exchange                 Deploy an exchange contract.
+  identity-implementation  Deploy an identity implementation contract.
+  identity-proxy-factory   Deploy an identity proxy factory.
+  test                     Deploy contracts for testing.
```

Comment:
Why's that weird coloured?

Reply:
Something with the bash formatter. I removed it.